low-level/framework/base/sspl_constants.py

```diff
@@ -116,22 +116,35 @@
 salt_uniq_attr_per_node = ['cluster_id']
 salt_uniq_passwd_per_node = ['RABBITMQINGRESSPROCESSOR', 'RABBITMQEGRESSPROCESSOR', 'LOGGINGPROCESSOR']
 
-try:
-    setup_info = subprocess.Popen(['sudo', '/usr/bin/provisioner', 'get_setup_info'],
-        stdout=subprocess.PIPE).communicate()[0].decode("utf-8").rstrip()
-    setup_info = ast.literal_eval(setup_info)
-    storage_type = setup_info['storage_type'].lower()
-    server_type = setup_info['server_type'].lower()
-    logger.info(f"Storage Type : '{storage_type}'")
-    logger.info(f"Server Type '{server_type}'")
+# Provisioner API Call Keys
+NODE_KEY = 'nodes'
+SERVER_PER_NODE_KEY = 'servers_per_node'
+STORAGE_TYPE_KEY = 'storage_type'
+SERVER_TYPE_KEY = 'server_type'
+STORAGE_TYPE = 'physical'
+SERVER_TYPE = 'rbod'
+
 
+try:
+    print('fetching the setup values:')
+    process = subprocess.Popen(['sudo', '/usr/local/bin/provisioner', 'get_setup_info'], \
```

Comment:
During consul vip change we had moved such stuff to a utility file since it is not a good idea to have it in this file. Can we move this code too to a utility file?

Reply:
Right. But problem is, that file gets packed into binary and python path gets set to framework directory which is inside binary. Now, this file gets installed outside binary (and inside binary also I think). And It gets executed at init step. And at that stage, it doesn't find that path which is inside binary. That is why we can't use that Utility class here !